docs/TranslationGuide.md

```diff
@@ -618,6 +620,14 @@ Modules are currently:
 - `sceneModelPosition`: Label used for the model position
 - `sceneModelRotation`: Label used for the model rotation
 - `sceneModelScale`: Label used for the model scale
+- `sceneModelMeterScale`: Label for metadata on meter scale for this model (how many units per metric meter)
+  - {0} the actual scale amount
+- `sceneModelLODLayers`: Label for metadata on number of LOD layers for this model
+  - {0} the number of LOD layers in this model
+- `sceneModelResolution`: Label for metadata on resolution at time of conversion for this model
+  - {0} the resolution of the model
+- `sceneModelAttributes: Label for metadata on number of attributes for this model
```

Comment:
No grave closes here

Reply:
I was wondering why all of the formatting died, nice find!